app/javascript/packs/mixins/validators_mixins.js

```diff
@@ -39,7 +39,9 @@ Vue.mixin({
     },
 
     avatar_url(validator) {
-      if(validator['avatar_url']) {
+      if(validator['avatar_file_url']) {
+        return validator['avatar_file_url']
+      } else if (validator['avatar_url']) {
```

Comment:
I think the whole point of this story was to not display original URL addresses. So we should not display avatar_url, only avatar_file_url or default image.

Reply:
true, but the images won't download instantly, so I thought we could leave it for the transition period

Reply:
Makes sense, I'll create a follow-up story to remove them